common/build-tools/src/main/resources/suppressions.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
```

Comment:
As part of the clean-up these should be moved the suppression files under the eng directory.